### ISSUE TITLE
Clean theme targets should meet minimum size

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5276]
Clean theme target size is not always compliant with target size minimum from WCAG 2.5.8.  See #5156.

Tests run for the audit found the following, however this is illustrative as tests should be run at time of development work to ensure all targets meet requirement.

| Page                | a  | button | input | ul | Total Failures|
| --------------------------- | -- | ------ | ----- | -- | ----------- |
| olh/accessibility/          |    | 1      |       |    | 1           |
| olh/search/                 |    |        | 6     |    | 6           |
| olh/collections/            |    | 3      |       |    | 3           |
| olh/news/429/               |    | 1      |       |    | 1           |
| olh/articles/               |    |        | 56    |    | 56          |
| olh/article/id/4405/        |    | 6      |       |    | 6           |

**Comment by ajrbyers** [comment]
Closed by https://github.com/openlibhums/janeway/pull/5553
